### ISSUE TITLE
Add new render function for product block component for `write to RIPE DB` resource type

### ISSUE THREAD

**Issue by wouter1975** [workfloworchestrator/orchestrator-ui-library#1635]
See https://git.ia.surfsara.nl/netdev/automation/projects/orchestrator-ui/-/issues/56 -- open-api url is here

waiting for BE ticket https://git.ia.surfsara.nl/netdev/automation/projects/ipam/-/issues/48 (closed)

As an example

<img width="499" alt="Image" src="https://github.com/user-attachments/assets/2a127b51-10b9-4944-8a75-9d156fb885c0" />

The IP subnet can be extracted from the subscription description, and the network address and broadcast address can be calculate. Or use the IPAM call and filter on prefix.

- Overwrite function similar to IPAM

- [x] We need to get the production api key before we can deploy this to production. --> not needed for GET calls

- [x] Igor fix the inet call to support x.x.x.x/x formatting

New api url:
`'https://igor.test.automation.surf.net/api/surf/ripe/inet/:ipam_prefix_id/2'`
curl -X ‘GET’ \
  ‘https://igor.test.automation.surf.net/api/surf/ripe/inet/:ipam_prefix_id/2’ \
  -H ‘accept: application/json; charset=utf-8’ \
  -H ‘Authorization: Bearer ...’

Alternative:
https://ricardo-1635-2.test.automation.surf.net/api/surf/ripe/inet/:ipam_prefix_id/1

WAITING for:
- [ ] Wait for backend to be in Production (https://git.ia.surfsara.nl/netdev/automation/projects/ripe-proxy/-/issues/1)
- [x] New endpoint in backend to provide the information -- to simulate on DEV use prfix_id `1`
